diff --git a/libero/__init__.py b/libero/__init__.py
new file mode 100644
index 0000000..e69de29
diff --git a/libero/libero/benchmark/__init__.py b/libero/libero/benchmark/__init__.py
index 2b833d0..5734edb 100644
--- a/libero/libero/benchmark/__init__.py
+++ b/libero/libero/benchmark/__init__.py
@@ -161,7 +161,7 @@ class Benchmark(abc.ABC):
             self.tasks[i].problem_folder,
             self.tasks[i].init_states_file,
         )
-        init_states = torch.load(init_states_path)
+        init_states = torch.load(init_states_path, weights_only=False)
         return init_states
 
     def set_task_embs(self, task_embs):
diff --git a/libero/lifelong/evaluate.py b/libero/lifelong/evaluate.py
index d4db4dd..6f0e77f 100644
--- a/libero/lifelong/evaluate.py
+++ b/libero/lifelong/evaluate.py
@@ -251,7 +251,7 @@ def main():
         init_states_path = os.path.join(
             cfg.init_states_folder, task.problem_folder, task.init_states_file
         )
-        init_states = torch.load(init_states_path)
+        init_states = torch.load(init_states_path, weights_only=False)
         indices = np.arange(env_num) % init_states.shape[0]
         init_states_ = init_states[indices]
 
diff --git a/libero/lifelong/metric.py b/libero/lifelong/metric.py
index 9777577..f8bccf2 100644
--- a/libero/lifelong/metric.py
+++ b/libero/lifelong/metric.py
@@ -104,7 +104,7 @@ def evaluate_one_task_success(
         init_states_path = os.path.join(
             cfg.init_states_folder, task.problem_folder, task.init_states_file
         )
-        init_states = torch.load(init_states_path)
+        init_states = torch.load(init_states_path, weights_only=False)
         num_success = 0
         for i in range(eval_loop_num):
             env.reset()
diff --git a/libero/lifelong/utils.py b/libero/lifelong/utils.py
index c0c3f10..0648af6 100644
--- a/libero/lifelong/utils.py
+++ b/libero/lifelong/utils.py
@@ -56,7 +56,7 @@ def torch_save_model(model, model_path, cfg=None, previous_masks=None):
 
 
 def torch_load_model(model_path, map_location=None):
-    model_dict = torch.load(model_path, map_location=map_location)
+    model_dict = torch.load(model_path, map_location=map_location, weights_only=False)
     cfg = None
     if "cfg" in model_dict:
         cfg = model_dict["cfg"]